src/main/resources/swagger/ga4gh-tool-discovery.yaml

```diff
@@ -172,7 +166,7 @@ paths:
           schema:
             $ref: '#/definitions/Error'
 
-  /tools/{id}/versions/{version-id}/{type}/descriptor/{relative-path}:
+  /tools/{id}/versions/{version_id}/{type}/descriptor/{relative_path}:
```

Comment:
@agduncan94  For this update, let's reach out to cwltool and determine how they want to tackle https://github.com/ga4gh/dockstore/issues/844

Reply:
Looks like we're just going to modify UI2's launch command to use the main descriptor as the path parameter of the relative paths endpoint